ansible/roles/vault_utils/tasks/push_secrets.yaml

```diff
@@ -158,3 +189,12 @@
   loop_control:
     label: "{{ item.key }}"
   when: not debug | default(False) | bool
+
+- name: Add the file secrets to the vault
+  ansible.builtin.shell:
```

Comment:
Maybe just add # noqa command-instead-of-shell so we can remove the warn_list: in ansible-lint?